odl/space/base_ntuples.py

```diff
@@ -493,12 +493,18 @@ def __init__(self, size, dtype):
             self.__is_real = True
             self.__real_dtype = self.dtype
             self.__real_space = self
-            self.__complex_dtype = TYPE_MAP_R2C.get(self.dtype, None)
+            try:
+                self.__complex_dtype = complex_dtype(self.dtype)
```

Comment:
No, that was the reason I had the `use_default` thing before. Now if no dtype is found and `default=None`, it's an error. If we want that behavior, then okay.